src/shared/dbus-client.c

```diff
@@ -327,22 +327,36 @@ dbus_bool_t bluealsa_dbus_get_pcm(
 	dbus_bool_t rv = TRUE;
 	size_t length = 0;
 	size_t i;
+	dbus_bool_t get_last = bacmp(addr, BDADDR_ANY) == 0;
```

Comment:
Use real `bool` type. The `dbus_bool_t` above is required because `dbus_bool_t != bool` and I wanted `bluealsa_dbus_` API to be compatible with libdbus-1.

Also, maybe it will be a good idea to add 00:00:00:00:00:00 as a default MAC in the `20-bluealsa.conf` file?

Reply:
done